packages/figma-block/src/settings.ts

```diff
@@ -71,12 +72,20 @@ export const settings: BlockSettings = {
         },
     ],
     layout: [
+        {
+            id: HAS_FUNCTIONALITY_ID,
+            type: 'switch',
+            label: 'Functionality',
+            defaultValue: false,
+            info: 'This is much more fun :)',
```

Comment:
Is the tooltip text like that in the design? 😂

Reply:
You are to fast this time :), will let you know when it is ready, I am waiting for this :) but thnx for checking the PR